services/ark-dashboard/ark-dashboard/lib/services/memory.ts

```diff
@@ -115,6 +117,8 @@ export const memoryService = {
     queryId: string;
     message: { role: string; content: string; name?: string };
     sequence?: number;
+    agent_prompt?: string;
```

Comment:
worth double checking this cause i don't think we actually store / use the prompt